src/quest_manager/models.py

```diff
@@ -954,13 +958,38 @@ def is_returned(self):
     def get_comments(self):
         return Comment.objects.all_with_target_object(self)
 
+    def _fix_ordinal(self):
+        # NOTE: There is a rare bug that we are unable to reproduce as of the moment where a QuestSubmission has the same ordinal.
+        # This is just a temporary hack where it will automatically fix the incorrect ordinal
+        broken_ordinal_start = self.ordinal - 1
+        submissions = QuestSubmission.objects.filter(quest=self.quest, user=self.user, ordinal__gte=self.ordinal - 1).order_by('time_completed')
+        logger.debug(
```

Comment:
If my memory is correct, this should be logged in one of the log files.

It seems that this will never be called since `LOGGING` is not configured here.. though this could provide useful information so that we could see which quest submissions were having incorrect ordinals